woocommerce/woocommerce
# Add/a2 p create settings crud api core tests

### All Submissions:






### Changes proposed in this Pull Request:
Add playwright api-core-tests for settings crud operations




Closes 409-gh-woocommerce/woocommerce-quality and 413-gh-woocommerce/woocommerce-quality.






### How to test the changes in this Pull Request:



1. Pull down the code
2. Navigate to woocommerce/plugins/woocommerce
3. Execute tests with USE_WP_ENV=1 pnpm playwright test --config=tests/api-core-tests/playwright.config.js tests/api-core-tests/tests/settings/settings-crud.test.js  
4. The settings tests should execute and pass successfully



### Other information:




### FOR PR REVIEWER ONLY:
